src/test/java/org/sagebionetworks/bridge/sdk/integration/HealthDataTest.java

```diff
@@ -212,12 +224,11 @@ public void canGetHealthDataByDateRange() {
         }
     }
 
-    private HealthDataRecord makeHDR(long version, String guid, DateTime startDate, DateTime endDate, JsonNode data) {
+    private HealthDataRecord makeHDR(String guid, DateTime startDate, DateTime endDate, JsonNode data) {
         HealthDataRecord record = new HealthDataRecord();
         record.setData(data);
         record.setStartDate(startDate);
         record.setEndDate(endDate);
-        record.setVersion(version);
         return record;
     }
```

Comment:
It looks like this method takes guid but just throws it away. Should it be calling setGuid() or does it not need it?